mastermindview/src/main.js

```diff
@@ -2,6 +2,29 @@ import Vue from 'vue'
 import App from './App.vue'
 import router from './router'
 import store from './store'
+import VueNotifications from "vue-notifications";
+import miniToastr from "mini-toastr";
+const toastTypes = {
+  success: "success",
+  error: "error",
+  info: "info",
+  warn: "warn"
+};
+
+miniToastr.init({ types: toastTypes });
+const toast = ({ title, message, type, timeout, cb }) => {
+  return miniToastr[type](message, title, timeout, cb);
```

Comment:
Als ik kijk op de npm site bij mini-toastr doen zij het met miniToastr.info(message, title, timeout, cb);. Ik denk niet dat dit werkt want, wat je nu doet is een element uit een array opvragen ofzo.